notification/app/notification/services/SloTrackingSender.scala

```diff
@@ -0,0 +1,24 @@
+package notification.services
+
+import aws.AWSAsync
+import com.amazonaws.services.sqs.AmazonSQSAsync
+import com.amazonaws.services.sqs.model.SendMessageRequest
+import org.slf4j.{Logger, LoggerFactory}
+
+import java.util.UUID
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+class SloTrackingSender(sqsClient: AmazonSQSAsync, queueUrl: String)(implicit ec: ExecutionContext) {
+  implicit private val logger: Logger = LoggerFactory.getLogger(this.getClass)
+  def sendTrackingMessage(notificationId: UUID): Unit = {
+    val sendMessageRequest: SendMessageRequest = new SendMessageRequest()
+      .withQueueUrl(queueUrl)
+      .withMessageBody(notificationId.toString)
+      .withDelaySeconds(600) // This message will become visible to queue consumers after 10 minutes
```

Comment:
For more details on this feature see [these docs](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-message-timers.html).